### DISCUSSION TITLE
Question: Is there a script that will trigger Summon 1?

### DISCUSSION THREAD

**Discussion by ZaViper** [sfmict/MountsJournal#37] [Q&A] [answered]
I know losing /click sucks. I really miss the macro command /click MountsJournal_Mount. I was wondering if there was a script like /run (something, something) that will achieve the same result as /click MountsJournal_Mount once did.

**Comment by sfmict** [comment, marked as answer]
Hi, unfortunately blizz removed that behavior :(, I can't do something.
